src/search.c

```diff
@@ -49,12 +49,19 @@ volatile int ABORT_SIGNAL; // Global ABORT flag for threads
 volatile int IS_PONDERING; // Global PONDER flag for threads
 volatile int ANALYSISMODE; // Whether to make some changes for Analysis
 
+int LateMovePruningCounts[2][9];
+
 void initSearch() {
 
     // Init Late Move Reductions Table
     for (int depth = 1; depth < 64; depth++)
         for (int played = 1; played < 64; played++)
             LMRTable[depth][played] = 0.75 + log(depth) * log(played) / 2.25;
+
+    for (int depth = 0; depth < 9; ++depth) {
+        LateMovePruningCounts[0][depth] = roundf((depth > 0 ? 2.5 : 0) + depth * depth * 2 / 4.5);
+        LateMovePruningCounts[1][depth] = roundf((depth > 0 ?   4 : 0) + depth * depth * 4 / 4.5);
```

Comment:
* The first line has ``2.5 : 0``, and the next has ``4 : 0``. Make the second line ``4.0 : 0`` to match the precision. 
* The end of each line has a 2x and a 4x multiplayer. Can you move that to the front, so it reads ``2 * depth * depth / 4.5`` to match other cases in the code if I recall.

Reply:
Change ``++depth`` to ``depth++`` since I always use ``i++`` in most cases in Ethereal

Reply:
Lets clean up the stack of includes/globals to be:

```
int LMRTable[64][64];                    
int LateMovePruningCounts[2][9]; 

volatile int ABORT_SIGNAL; // Global ABORT flag for threads
volatile int IS_PONDERING; // Global PONDER flag for threads
volatile int ANALYSISMODE; // Whether to make some changes for Analysis
```

I dropped the comment from LMRTable. LMR is ubiquitous. LMP, FMP, CMP, SNMP, those are not. but in this case we have Late Move Pruning fully typed out.